.changeset/big-trains-learn.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Add `ignoreSelectors` option to `selector-max-compound-selectors` rule
```

Comment:
[suggest] Let's follow the pull request title:

```suggestion
Added: `ignoreSelectors: []` to `selector-max-compound-selectors`
```

Reply:
As long as `Add` is corrected I am good.

```suggestion
Added: `ignoreSelectors` option to `selector-max-compound-selectors` rule
```

would be acceptable too.

Reply:
For consistency. See:

https://github.com/stylelint/stylelint/blob/b6446093297c9a6a147c3c967067f06c12d76ab1/CHANGELOG.md?plain=1#L16